.github/scripts/aur-preflight.sh

```
@@ -0,0 +1,180 @@
+#    Match the whole value: a 64 char string with one stray character is not a
+#    digest, and makepkg would reject it after the user has downloaded 75 MB.
+first_sum="${sha256sums[0]-}"
+if [[ ! "$first_sum" =~ ^[0-9a-f]{64}$ ]]; then
+    note "sha256sums[0] is '${first_sum}', expected 64 lowercase hex characters (the .deb digest)"
+fi
+
+# 4. Both source lines are expanded in .SRCINFO, so a hand edit can update one and
+#    miss the other, and a URL that hardcodes a version ignores the bump entirely.
+for s in "${source[@]}" "${srcinfo_sources[@]}"; do
+    case "$s" in
+        *"$pkgver"*) ;;
```

Shellcheck (0.11.0)

[warning] pkgver is referenced but not assigned.

(SC2154)